### ISSUE TITLE
New Rule Proposal: HTML placeholder attribute  used incorrectly (it duplicates the accessible name for the control)

### ISSUE THREAD

**Issue by spanchang** [act-rules/act-rules.github.io#965]
Impact: Not a WCAG violation but results in repetitive reading of identical text for screen reader users, degrading user experience.

Test process:
Check if the text content for placeholder attribute  is identical to the form control's accessible name i.e. label text   

Remediation:
Do away with the placeholder attribute completely if there is no hint or advisory text  that needs to be provided for the control. 
Else ensure the placeholder is used as per HTML5 specs.
HTML5 password attribute is meant to provide a hint or advisory text for the control and not function as a label.

**Comment by jkodu** [comment]
@spanchang 

We had a discussion on this rule idea & would like to proceed with this. Would you be able to come up with a pull request for the rule?

**Comment by WilcoFiers** [comment]
Going to close this issue, as it does not seem like anyone is taking this up.
